### PR TITLE
[Word] (style) Include BorderWidth

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
Fixes https://github.com/OfficeDev/office-js-docs-reference/issues/1731

### CODE REVIEW

File: samples/word/99-preview-apis/manage-styles.yaml [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+            <h3>Try it out</h3>
+            <p>
+                <h4>Get current number of styles</h4>
+                <button id="count" class="ms-Button">
+                        <span class="ms-Button-label">Get count</span>
+                </button>
+            </p>
+
+            <h4>Add a new style</h4>
+            <div>Name the style using letters. Can include digits. Examples: NewName, newname1</div>
```
Comment (AlexJerabek) [inline samples/word/99-preview-apis/manage-styles.yaml:267]:
```suggestion
            <div>Give the style an alphanumeric name. Examples: NewName, newname1</div>
```

File: samples/word/99-preview-apis/manage-styles.yaml [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+        <section class="samples ms-font-m">
+            <h3>Try it out</h3>
+            <p>
+                <h4>Get current number of styles</h4>
+                <button id="count" class="ms-Button">
+                        <span class="ms-Button-label">Get count</span>
+                </button>
+            </p>
+
+            <h4>Add a new style</h4>
```
Comment (AlexJerabek) [inline samples/word/99-preview-apis/manage-styles.yaml:266]: I think "new" is implied by adding something. Adding "custom" here helps imply all the other actions are related.
```suggestion
            <h4>Add a custom style</h4>
```

File: samples/word/99-preview-apis/manage-styles.yaml [hunk 3]
```diff
@@ -0,0 +1,357 @@
...
+                <p>Select a Word document to get its style info.</p>
+                <form>
+                    <input type="file" id="file" />
+                </form>
+                    <br>
+                    <p>Get style info from the selected document.</p>
+                    <button id="get-external-styles" class="ms-Button">
+                      <span class="ms-Button-label">Get styles</span>
+                    </button>
+        </section>
```
Comment (AlexJerabek) [inline samples/word/99-preview-apis/manage-styles.yaml:334]: Two things here:

1. Consider making the external file stuff a separate sample. It seems distinct enough.
2. The indentation in the HTML seems off.

File: samples/word/99-preview-apis/manage-styles.yaml [hunk 4]
```diff
@@ -0,0 +1,357 @@
...
+
+            if (style.isNullObject) {
+              console.warn(`There's no existing style with the name '${styleName}'.`);
+            } else if (style.type != Word.StyleType.paragraph) {
+              console.log(`The '${styleName}' style isn't a paragraph style.`);
+            } else {
+              const body = context.document.body;
+              body.clear();
+              body.insertParagraph(
+                "Video provides a powerful way to help you prove your point. When you click Online Video, you can paste in the embed code for the video you want to add. You can also type a keyword to search online for the video that best fits your document.",
```
Comment (AlexJerabek) [inline samples/word/99-preview-apis/manage-styles.yaml:106]: I've always found this text about video kind of weird considering you can't add video through the JS APIs.

File: samples/word/99-preview-apis/manage-styles.yaml [hunk 5]
```diff
@@ -0,0 +1,357 @@
+id: word-manage-styles
+name: Manage styles
+description: This sample shows how to perform operations on the styles in the current document and how to add and delete custom styles.
```
Comment (AlexJerabek) [inline samples/word/99-preview-apis/manage-styles.yaml:3]: This doesn't mention loading styles (from other files).

File: samples/word/99-preview-apis/manage-styles.yaml [hunk 6]
```diff
@@ -0,0 +1,357 @@
...
+        $("#apply-style").click(() => tryCatch(applyStyle));
+        $("#font-properties").click(() => tryCatch(setFontProperties));
+        $("#paragraph-format").click(() => tryCatch(setParagraphFormat));
+        $("#border-properties").click(() => tryCatch(setBorderProperties));
+        $("#delete-style").click(() => tryCatch(deleteStyle));
+        $("#file").change(getBase64);
+        $("#get-external-styles").click(() => tryCatch(getExternalStyles));
+
+        async function getCount() {
+          // Gets the number of styles.
```
Comment (AlexJerabek) [inline samples/word/99-preview-apis/manage-styles.yaml:21]: Consider making it more evident this is about the number of styles available, not actively used.
```suggestion
          // Gets the number of styles stored with the document.
```

**Review by AlexJerabek — APPROVED** [review]
Approving with a couple suggestions. If you take the suggestion to split this into two samples, I can take another look.

File: samples/word/99-preview-apis/manage-styles.yaml (hunk 4 above)
Comment (ElizabethSamuel-MSFT) [reply samples/word/99-preview-apis/manage-styles.yaml:106]: Do you want us to change the setup text to use "Lorem ipsum" or some other content?

File: samples/word/99-preview-apis/manage-styles.yaml (hunk 1 above)
Comment (ElizabethSamuel-MSFT) [reply samples/word/99-preview-apis/manage-styles.yaml:267]: What about this?

```suggestion
            <div>Give the style an alphanumeric name. Must start with a letter. Examples: NewName, newname1</div>
```

File: samples/word/99-preview-apis/manage-styles.yaml (hunk 4 above)
Comment (AlexJerabek) [reply samples/word/99-preview-apis/manage-styles.yaml:106]: Not sure if it's worth the effort, but something to consider in a future sample.

File: samples/word/99-preview-apis/manage-styles.yaml (hunk 1 above)
Comment (AlexJerabek) [reply samples/word/99-preview-apis/manage-styles.yaml:267]: Yeah, that works!

**Review by AlexJerabek — APPROVED** [review]

**Pull request merged**